datastore/datastore.py

```diff
@@ -236,50 +235,52 @@ def get_similar_dictionary(self, entity_name, text, fuzziness_threshold="auto:4,
             db.get_similar_ngrams_dictionary(entity_name='city', ngrams_list=ngrams_list, fuzziness_threshold=2)
 
             Output:
-                {u'Bangalore': u'Bangalore',
-                 u'Mulbagal': u'Mulbagal',
-                 u'Multai': u'Multai',
-                 u'Mumbai': u'Mumbai',
-                 u'Pune': u'Pune',
-                 u'Puri': u'Puri',
-                 u'bangalore': u'bengaluru',
-                 u'goa': u'goa',
-                 u'mumbai': u'mumbai',
-                 u'pune': u'pune'}
-        """
-        results_dictionary = collections.OrderedDict()
+                [
+                    {u'Bangalore': u'Bangalore',
+                     u'Mulbagal': u'Mulbagal',
+                     u'Multai': u'Multai',
+                     u'Mumbai': u'Mumbai',
+                     u'Pune': u'Pune',
+                     u'Puri': u'Puri',
+                     u'bangalore': u'bengaluru',
+                     u'goa': u'goa',
+                     u'mumbai': u'mumbai',
+                     u'pune': u'pune'}
+                 ]
+        """
+        results_list = []
         if self._client_or_connection is None:
             self._connect()
         if self._engine == ELASTICSEARCH:
             self._check_doc_type_for_elasticsearch()
             request_timeout = self._connection_settings.get('request_timeout', 20)
-            results_dictionary = elastic_search.query.full_text_query(connection=self._client_or_connection,
-                                                                      index_name=self._store_name,
-                                                                      doc_type=self._connection_settings[
-                                                                          ELASTICSEARCH_DOC_TYPE],
-                                                                      entity_name=entity_name,
-                                                                      sentence=text,
-                                                                      fuzziness_threshold=fuzziness_threshold,
-                                                                      search_language_script=search_language_script,
-                                                                      request_timeout=request_timeout,
-                                                                      **kwargs)
-        return results_dictionary
+            results_list = elastic_search.query.full_text_query(connection=self._client_or_connection,
+                                                                index_name=self._store_name,
+                                                                doc_type=self._connection_settings[
+                                                                    ELASTICSEARCH_DOC_TYPE],
+                                                                entity_name=entity_name,
+                                                                sentence=text,
```

Comment:
- Use `texts` as argument name on Line 233. 
- Use `sentences` as argument for `elastic_search.query.full_text_query`.

Makes it clear that this function takes in list